www-client/netsurf/netsurf-3.3.recipe

```diff
@@ -9,65 +9,70 @@ developed, NetSurf is continually evolving and improving."
 HOMEPAGE="http://www.netsurf-browser.org/"
 COPYRIGHT="2003-2014 The NetSurf Browser project"
 LICENSE="GNU GPL v2"
-REVISION="2"
+REVISION="3"
 # Do NOT use the netsurf-all package. It contains a bunch of subprojects which
 # we build as separate packages so they can easily be used by other projects.
 SOURCE_URI="http://download.netsurf-browser.org/netsurf/releases/source/netsurf-$portVersion-src.tar.gz"
 CHECKSUM_SHA256="23f92a0940f577b9605f2e54786fea3521680d32d6c00f2a3db1eae3f29e3686"
-PATCHES="netsurf-$portVersion.patchset"
+
+if [ $targetArchitecture != x86 ]; then
+	PATCHES="netsurf-$portVersion.patchset"
+else
+	PATCHES="netsurf_x86-$portVersion.patchset"
+fi
```

Comment:
I¡s this required? As far as I know the changes on each side should not interfere with each other so I would just put everything in a single common patchset.